app/views/peeps/index.erb

```diff
@@ -0,0 +1,12 @@
+<h1>Peeps</h1>
+<ul id="peeps">
+  <% @peeps.reverse.each do |peep| %>
```

Comment:
If you have to reverse the peeps every time, you could have a class method in your peeps class that returns the all the tweets in reverse order. This would have the benefit of reducing the amount of code in both your view and your controller.